### DISCUSSION TITLE
Permanent shared link

### DISCUSSION THREAD

**Discussion by chilikasha** [antonmedv/numbr#7] [Ideas]
Hi,

Currently whenever a change is made in a doc, a new link is generated, and previously shared or bookmarked link doesn't make sense. Is it (or would it be) possible to have a permanent link?

Thank you!

**Comment by antonmedv** [comment]
I’m working on new backend, which will allow to have something similar persistent docs.

**Reply by chilikasha** [reply]
Glad to hear that, thanks again
